vtex/faststore
# Feat: Next 13 - Global Section - migrate `CartSidebar` Section - SFS-1014

## What's the purpose of this pull request?

This PR aims to migrate the `CartSidebar`  from the pages router to the app router (next 13). 

## How to test it?

- Run the local server (`yarn dev` in the core package);
- Make sure that all the current pages are working as expected (`/`, `/office` and some `/<slug>/p`);
- Make sure `/fs-next-update` is rendering without errors with the `CartSidebar` section at the top of the page.


<img width="323" alt="Screenshot 2024-06-14 at 12 31 10" src="https://github.com/vtex/faststore/assets/11325562/e263a79c-cd2f-46e4-8eef-b215037085cb">
<img width="321" alt="Screenshot 2024-06-14 at 12 31 24" src="https://github.com/vtex/faststore/assets/11325562/4a527c4d-40c1-4843-a9f7-634ee833ad31">
